.github/workflows/ci.yml

```
@@ -49,9 +49,28 @@ jobs:
           cargo audit
       
       - name: Run tests
-        run: cargo test --workspace --verbose
+        run: |
+          # Run unit and integration tests
+          cargo test --workspace --verbose
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)